packages/server/__tests__/loginWithPassword.test.ts

```diff
@@ -58,4 +59,7 @@ test('Login after signup', async () => {
       }
     }
   })
+  const lastSeenAt = Date.parse(login.data.loginWithPassword.user.lastSeenAt)
```

Comment:
+2 i've never seen the `parse` method before, i always used new Date().getTime(). TIL